src/modules/pair/models/pair.compounded.apr.model.ts

```diff
@@ -0,0 +1,26 @@
+import { ObjectType, Field } from '@nestjs/graphql';
+
+@ObjectType()
+export class PairCompoundedAPRModel {
```

Comment:
you can move this model into pair.model.ts file. No need to have it on different file

Reply:
done